Eplant/View/index.ts

```diff
@@ -18,6 +18,7 @@ type ViewAction<State, Action> = {
 }
 
 export interface View<Data = any, State = any, Action = any> {
+  [x: string]: any
```

Comment:
What is this line for?

Reply:
It's probably better to add a field that's a dictionary, instead of making the view itself be one